src/utils/ParameterConstraintsUtils.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+const operatorsDict = {
+  date: ['>', '<', '>=', '<=', '==', '!='],
+  number: ['>', '<', '>=', '<=', '==', '!='],
+  int: ['>', '<', '>=', '<=', '==', '!='],
+  string: ['==', '!='],
+  bool: ['==', '!='],
+};
+
+const getIsVarTypesComparisonValid = (varType, varTypeToCompare) => {
+  const numberVarTypes = ['int', 'number'];
+  return varType === varTypeToCompare || [varType, varTypeToCompare].every((el) => numberVarTypes.includes(el));
+};
+
+const getParameterValidationConstraint = (validationString, varType, parametersList) => {
+  if (validationString) {
+    const [operator, id] = validationString.replace(/\s+/g, ' ').trim().split(' ');
+    if (id === undefined) return null;
```

Comment:
What about _id_ is equal to _null_ ?

Reply:
`split` function cannot return `null`, it will return `undefined` (if there is no spaces in validation string for example: `validation: '>parameter'`) otherwise it will be a string